app/views/posts/index.html.erb

```diff
@@ -1,2 +1,82 @@
-<h1>Posts#index</h1>
-<p>This is going to show all posts from a given user</p>
+<section class="show-posts">
+  <div class="user-container">
+    <div class="picture">
+      <div class="empty-div">
+      </div>
+    </div>
+    <div class="user-name">
+        <h1>
+          Name
+        </h1>
+        <p>
+          Number os posts: X
+        </p>
+    </div>
+  </div>
+  <div class="post-1 post">
+    <div class="post-info">
+      <h4 class="post-title">
+        Post title
+      </h4>
+      <p class="post-text">
+        Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua sed do eiusmod tempor incididunt ut labore et dolore magna aliqua sed do eiusmod tempor incididunt ut labore et dolore magna aliqua sed do eiusmod tempor incididunt ut labore et dolore magna aliqua...
+      </p>
+      <div class="comms-likes">
+        <p class="comments-counter">
+          Comments: X
+        </p>
+        <p class="likes-counter">
+          Likes: Y
+        </p>
+      </div>
+    </div>
+    <div class="comments-post-1 comments">
+      <div>
+        <p>Username: Comment 1</p>
+      </div>
+      <div>
+        <p>Username: Comment 2</p>
+      </div>
+      <div>
+        <p>Username: Comment 3</p>
+      </div>
+      <div>
+        <p>Username: Comment 4</p>
+      </div>
+    </div>
+  </div>
+  <div class="post-2 post">
+    <div class="post-info">
+      <h4 class="post-title">
+        Post title
+      </h4>
+      <p class="post-text">
+        Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua sed do eiusmod tempor incididunt ut labore et dolore magna aliqua sed do eiusmod tempor incididunt ut labore et dolore magna aliqua...
+      </p>
+      <div class="comms-likes">
+        <p class="comments-counter">
+          Comments: X
+        </p>
+        <p class="likes-counter">
+          Likes: Y
+        </p>
+      </div>
+    </div>
+    <div class="comments-post-2 comments">
+      <div>
+        <p>Username: Comment 1</p>
+      </div>
+      <div>
+        <p>Username: Comment 2</p>
+      </div>
+      <div>
+        <p>Username: Comment 3</p>
+      </div>
+      <div>
+        <p>Username: Comment 4</p>
+      </div>
+    </div>
+  </div>
+
+  <a href="posts/show">Pagination</a>
+</section>
```

Comment:
You are not using the method `post.recent_comments`, please use this method to render 5 most recent comments. I read your text on the `Additional Information` and in order for you to add Posts without having a view, you can use the rails console.

- To create a post without a view you can use the rails console, you can check this [link](https://guides.rubyonrails.org/command_line.html) to the rails guide to have more info about it.